runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/artifact/BeamFileSystemArtifactRetrievalService.java

```diff
@@ -182,6 +182,10 @@ static ProxyManifest loadManifest(String retrievalToken) throws IOException {
     LOG.info("Loading manifest for retrieval token {}", retrievalToken);
     // look for manifest file at $retrieval_token
     ResourceId manifestResourceId = getManifestLocationFromToken(retrievalToken);
+    return loadManifest(manifestResourceId);
+  }
+
+  public static ProxyManifest loadManifest(ResourceId manifestResourceId) throws IOException {
```

Comment:
Lets keep it package private as on BeamFileSystemArtifactRetricalService is the only other user.